frontend/src/component/personalDashboard/PersonalDashboard.tsx

```diff
@@ -285,7 +289,7 @@ export const PersonalDashboard = () => {
 
     const [welcomeDialog, setWelcomeDialog] = useLocalStorageState<
         'open' | 'closed'
-    >('welcome-dialog:v1', 'open');
+    >('welcome-dialog:v1', splash?.personalDashboardKeyConcepts ? 'closed' : 'open');
```

Comment:
Linter's complaining.


```suggestion
    >(
        'welcome-dialog:v1',
        splash?.personalDashboardKeyConcepts ? 'closed' : 'open'
    );
```